packages/components/src/components/hds/side-nav/index.ts

```diff
@@ -9,20 +9,64 @@ import { action } from '@ember/object';
 import { assert } from '@ember/debug';
 import { registerDestructor } from '@ember/destroyable';
 
-export default class HdsSideNavComponent extends Component {
+import type { HdsSideNavBaseSignature } from './base';
+
+interface HdsSideNavSignature {
+  Args: {
+    isResponsive?: boolean;
+    isCollapsible?: boolean;
+    isMinimized?: boolean;
+    hasA11yRefocus?: boolean;
+    a11yRefocusSkipTo?: string;
+    a11yRefocusSkipText?: string;
+    a11yRefocusNavigationText?: string;
+    a11yRefocusRouteChangeValidator?: string;
+    ariaLabel?: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    onToggleMinimizedStatus?: (arg: boolean) => void;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    onDesktopViewportChange?: (arg: boolean) => void;
+  };
+  Blocks: {
+    root?: [];
```

Comment:
@alex-ju I just noticed that we don't expose a `:root` block, we just consume it from the "base"
```suggestion
```